examples/demo-react-native/e2e/example.spec.js

```diff
@@ -6,6 +6,14 @@ describe('Example', () => {
   it('should have welcome screen', async () => {
     await expect(element(by.id('welcome'))).toBeVisible();
   });
+
+  it('open datePicker screen', async () => {
+    await element(by.id('datePickerIOS')).tap();
+    await expect(element(by.type('UIPickerView'))).toBeVisible();
+    await element(by.type('UIPickerView')).setColumnToValue(1,"6")
+    await element(by.type('UIPickerView')).setColumnToValue(2,"34")
```

Comment:
Let's add an `expect` here that checks that the correct date has been selected in the picker.